### PR TITLE
[Product Partnerships] Added Amazon Database Migration Service 

### PR DESCRIPTION
**Pull request by SuruchiSaboo**
[Product Partnerships] Added Amazon DMS Entity

## Pre merge checklist

- [X] Did you check you NRQL syntax? - Does it work?
- [X] Did you include a Data source and Documentation reference?
- [X] Are all documentation links publicly accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [X] Did you check your descriptive content for spelling and grammar errors?
- [X] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [X] Does the PR contain a screenshot for each of your dashboards?
- [X] Do your screenshots show data?
- [X] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [X] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/amazon-database-migration-service/LowFreeableMemory.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Low Freeable Memory
+
+description: |+
+  This alert occurs when Freeable Memory bytes go below 100 for 5 minutes
```
Comment (rahulbasu-nr) [inline alert-policies/amazon-database-migration-service/LowFreeableMemory.yml:4]: What is 100? Please specify the unit of measurement?

File: alert-policies/amazon-database-migration-service/LowFreeableMemory.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE 
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: BELOW
+    # Value that triggers a violation; float value
+    threshold: 100
```
Comment (rahulbasu-nr) [inline alert-policies/amazon-database-migration-service/LowFreeableMemory.yml:20]: Please specify the unit of measurement.

File: alert-policies/amazon-database-migration-service/LowFreeableMemory.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    # Time in seconds; 120 - 3600
+    thresholdDuration: 300
```
Comment (rahulbasu-nr) [inline alert-policies/amazon-database-migration-service/LowFreeableMemory.yml:22]: Please specify the unit of measurement.

File: alert-policies/amazon-database-migration-service/LowFreeableMemory.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    # How many data points must be in violation for the duration
+    thresholdOccurrences: ALL
+
+  # Adding a Warning threshold is optional
+  - priority: WARNING
+    operator: BELOW
+    threshold: 95
```
Comment (rahulbasu-nr) [inline alert-policies/amazon-database-migration-service/LowFreeableMemory.yml:29]: Please specify the unit of measurement.

File: quickstarts/aws/amazon-database-migration-service/config.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+documentation:
+  - name: Amazon Database Migration Service installation docs
+    description: |
+      Monitor Amazon Database Migration Service by connecting AWS to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-database-migration-service-monitoring-integration/
+keywords:
+  - aws
+  - amazon web services
+  - Database Migration Service
```
Comment (rahulbasu-nr) [inline quickstarts/aws/amazon-database-migration-service/config.yml:25]:
```suggestion

```

**Review by rahulbasu-nr — CHANGES_REQUESTED** [review]
Please make the changes.

File: alert-policies/amazon-database-migration-service/LowFreeableMemory.yml (hunk 1 above)
Comment (SuruchiSaboo) [reply alert-policies/amazon-database-migration-service/LowFreeableMemory.yml]: Done. Please review.

File: alert-policies/amazon-database-migration-service/LowFreeableMemory.yml (hunk 2 above)
Comment (SuruchiSaboo) [reply alert-policies/amazon-database-migration-service/LowFreeableMemory.yml:20]: Done. Unit specified in comment. Please review the changes.

File: alert-policies/amazon-database-migration-service/LowFreeableMemory.yml (hunk 3 above)
Comment (SuruchiSaboo) [reply alert-policies/amazon-database-migration-service/LowFreeableMemory.yml:22]: @rahulbasu-nr 
We don't have a separate field 'unit' under terms to specify the unit, we have been using comments to do that. I've made changes accordingly, please let me know if I need to make other changes.

File: alert-policies/amazon-database-migration-service/LowFreeableMemory.yml (hunk 4 above)
Comment (SuruchiSaboo) [reply alert-policies/amazon-database-migration-service/LowFreeableMemory.yml:30]: done, please review

File: quickstarts/aws/amazon-database-migration-service/config.yml (hunk 5 above)
Comment (SuruchiSaboo) [reply quickstarts/aws/amazon-database-migration-service/config.yml:25]: @rahulbasu-nr This Quickstart is for Database Migration Service, I don't understand why do we need to remove this keyword, could you please help me with this.

Comment (rahulbasu-nr) [reply quickstarts/aws/amazon-database-migration-service/config.yml:25]: The same keyword exists in line no. 27. So I suggested to remove line no. 25.

**Review by rahulbasu-nr — APPROVED** [review]
Looks good to me.

**Pull request merged**